pg_shard--1.0.sql

```diff
@@ -77,3 +77,78 @@ CREATE FUNCTION worker_copy_shard_placement(table_name text, source_node_name te
 RETURNS void
 AS 'MODULE_PATHNAME'
 LANGUAGE C STRICT;
+
+CREATE FUNCTION partition_column_to_node_string(table_oid oid)
+RETURNS text
+AS 'MODULE_PATHNAME'
+LANGUAGE C;
+
+COMMENT ON FUNCTION partition_column_to_node_string(oid)
+		IS 'return textual form of distributed table''s partition column';
+
+CREATE FUNCTION sync_table_metadata_to_citus(table_name text) RETURNS VOID
+AS $sync_table_metadata_to_citus$
+	DECLARE
+		table_relation_id CONSTANT oid NOT NULL := table_name::regclass::oid;
+		dummy_shard_length CONSTANT bigint := 0;
+	BEGIN
+		-- remove shard placements for table
```

Comment:
I'll remove the `DELETE` functionality. It was useful for me during testing, but I see it's either no necessary or incomplete for the end user.

I'll see about `CASCADE`. I didn't know Citus did that. What is the action here? Just to find out whether it happens?